libs/fixtures/data/electionFamousNames2021/cdf-election.ts

```diff
@@ -0,0 +1,244 @@
+        '@type': 'InternationalizedText',
+        Text: [
+          {
+            '@type': 'BallotDefinition.LanguageString',
+            Language: 'en',
+            Content: 'Lincoln Municipal General Election',
+          },
+        ], // Election name in other languages would go here
+      },
+
+      // Definition of all candidates in the election across contests
+      Candidate: [
+        {
+          '@type': 'BallotDefinition.Candidate',
+          '@id': 'sherlock-holmes',
+          BallotName: {
+            '@type': 'InternationalizedText',
+            Text: [
+              {
+                '@type': 'BallotDefinition.LanguageString',
+                Language: 'en',
+                Content: 'Sherlock Holmes',
+              },
+            ],
+          },
+          PartyId: 'party-0',
+        },
+        {
+          '@type': 'BallotDefinition.Candidate',
+          '@id': 'thomas-edison',
+          BallotName: {
+            '@type': 'InternationalizedText',
+            Text: [
+              {
+                '@type': 'BallotDefinition.LanguageString',
+                Language: 'en',
+                Content: 'Thomas Edison',
+              },
+            ],
+          },
+          PartyId: 'party-1',
+        },
+      ],
+
+      // Definition of contests
+      Contest: [
+        {
+          '@type': 'BallotDefinition.CandidateContest',
+          '@id': 'mayor',
+          ElectionDistrict: 'district-1',
+          // TODO what is the analogous field for "section"?
+          Name: 'Mayor', // Contest name, not for ballot
+          BallotTitle: {
+            // Display name for contest on ballot
+            '@type': 'InternationalizedText',
+            Text: [
+              {
+                '@type': 'BallotDefinition.LanguageString',
+                Language: 'en',
+                Content: 'Mayor',
+              },
+            ],
+          },
+          VotesAllowed: 1,
+          // TODO there's no field analogous to "allowWriteIns" in CDF
+          // Contest options reference the candidates defined above by id
+          ContestOption: [
+            {
+              '@type': 'BallotDefinition.CandidateOption',
+              CandidateIds: ['sherlock-holmes'],
+            },
+            {
+              '@type': 'BallotDefinition.CandidateOption',
+              CandidateIds: ['thomas-edison'],
+            },
+          ],
+        },
+      ],
+
+      // Definition of ballot styles
+      BallotStyle: [
+        {
+          '@type': 'BallotDefinition.BallotStyle',
+          '@id': 'ballot-style-1',
+          // TODO do we also reference districts here? what's the relationship between districts and precincts?
+          GpUnit: ['precinct-23', 'precinct-22'],
+        },
+      ],
+    },
+  ],
+
+  // Definition of parties
+  Party: [
+    {
+      '@type': 'BallotDefinition.Party',
+      '@id': 'party-0',
+      // TODO there's no way to distinguish between "name" and "fullName"
```

Comment:
In VX election def, we have a "name" and "fullName" field for a party. What are they used for? In CDF, we can only have one by default

Reply:
"Democratic" vs "Democratic Party", I think we originally used name but have moved to `fullName` as it supports localization better (e.g. "Democratic Party" --> "Partido Demócrata"). We still are using `name` in places but we probably want to drop it entirely. @benadida probably has most context here.

I think we should just use full name as we move to CDF.

Reply:
Sounds good to me